komodoenv/update.py

```diff
@@ -398,11 +402,12 @@ def main(args: List[str] = None):
 
     config = read_config()
     current = current_track(config)
-    if not should_update(config, current):
-        return
 
     check_same_distro(config)
 
+    if not should_update(config, current):
+        return
+
```

Comment:
This might be outside the current scope, but the function `check_same_distro` documentation/comment state that boolean is returned, but it does not do that.
Also, this will catch os-version mismatch?